### PR TITLE
fixed navbar, added property type garage cap

### PR DESCRIPTION
**Pull request by chougulesurajb**
fixed navbar refresh issue, 
added property type, garage capacity in the PropertyMeta.js

### CODE REVIEW

File: src/app/uploads/image-upload.js [hunk 1]
```diff
@@ -77,7 +77,9 @@ export const ImageUpload = () => {
               {fileNames.map((fileName) => (
                 <TableRow key={fileName}>
                   <TableCell style={{ width: "40%", padding: "20px" }}>
-                    {fileName.length > 20 ? fileName.slice(0, 20) + "..." : fileName}
+                    {fileName.length > 20
```
Comment (mchepuri) [inline src/app/uploads/image-upload.js:81]: No explicit changes besides justification  formatting changes

File: src/app/components/PropertyMeta.js [hunk 2]
```diff
@@ -1,92 +1,170 @@
...
+import { notifyDataUpdate } from "../util/util";
+import { useEffect, useState } from "react";
+import { viewAlbum } from "../uploads/core-sdk";
+const Photos = (props) => {
+  console.log("photos", props._photos);
+  const pics =
+    props._photos &&
+    props._photos.map((photo) => {
+      return <img src={photo} />;
     });
```
Comment (mchepuri) [inline src/app/components/PropertyMeta.js:17]: Nothing much changed except formatting till this point

File: src/app/components/PropertyMeta.js [hunk 3]
```diff
@@ -1,92 +1,170 @@
...
+            </MenuItem>
+          </Select>
+        </FormControl>
+      </form>
     </div>
-  }
-
-  
\ No newline at end of file
+  );
```
Comment (mchepuri) [inline src/app/components/PropertyMeta.js:169]: This looks good

File: src/app/uploads/image-upload.js (hunk 1 above)
Comment (chougulesurajb) [reply src/app/uploads/image-upload.js:81]: This is the auto-formatted by VS code.

File: src/app/components/PropertyMeta.js (hunk 2 above)
Comment (chougulesurajb) [reply src/app/components/PropertyMeta.js:17]: This is the auto-formatted by VS code.

File: src/app/components/PropertyMeta.js (hunk 3 above)
Comment (chougulesurajb) [reply src/app/components/PropertyMeta.js:169]: This is the auto-formatted by VS code.

**Pull request merged**